src/main/java/spoon/reflect/visitor/CtInheritanceScanner.java

```diff
@@ -106,8 +110,6 @@
  */
 public abstract class CtInheritanceScanner implements CtVisitor {
```

Comment:
You removed implementation of all scanners to force sub classes to implement appropriate methods. Good idea but can you explain this new practice in the javadoc?